source/documentation/fuseki2/fuseki-webapp.md

```diff
@@ -1,48 +1,71 @@
 ---
-title: Running Fuseki
+title: Running Fuseki with UI
 ---
 
-Fuseki can be run in a number of ways:
+Fuseki/UI can be run in a number of ways:
 
 * [As a standalone server](#fuseki-standalone-server)
 * [As a service](#fuseki-service) run by the operation system, for example, started when the machine
 * [As a Web Application](#fuseki-web-application) inside a container such as Apache Tomcat or Jetty
 boots.
-* [As a configurable SPARQL server](#fuseki-main)
-* [As a deployment and development standalone server](#fuseki-server)
+
+Fuseki is also packaged as a plain server ["Fuseki Main"](fuseki-main.html)
+with no UI for use as a configurable SPARQL server, for [building as a Docker
+container](fuseki-docker.html), and as a deployment and development standalone
+server.
+
+Both packaging used the same configuration file format, and in standalone server
+mode, the same command line arguments.
 
 See "[Fuseki Configuration](fuseki-configuration.html)" for information on
-how to provide datasets and configure services.
+how to provide datasets and configure services usoing the configuration file.
 
 ## Fuseki as a Standalone Server {#fuseki-standalone-server}
 
 This is running Fuseki from the command line.
 
-    fuseki-server [--mem | --loc=DIR] [[--update] /NAME]
-
-    fuseki-server --config=CONFIG
+To publish at <tt>http://<i>host</i>:3030/NAME</i></tt>:
 
 where `/NAME` is the dataset publishing name at this server in URI space.
 
+TDB1 database:
+
+    fuseki-server [--loc=DIR] [[--update] /NAME]
+
 The argument `--tdb2` puts the command line handling into "TDB2 mode".
 A dataset created with `--loc` is a TDB2 dataset.
+TDB2 database:
+
+    fuseki-server --tdb2 [--loc=DIR] [[--update] /NAME]
+
+In-memory, non-peristent database (always updatable):
+
+    fuseki-server --mem /NAME
+
+Load a file at start and provide it read-only:
+
+    fuseki-server --file=MyData.ttl /NAME
+
+where "MyData.ttl" can be any RDF format, both triples or quads. 
+
+Administrative functions are only avilable from "localhost".
```

Comment:
Huh, Today_I_Learned, thought the admin functions were always available, regardless of IP address/hostname :+1:

Reply:
Shiro!